src/xrpld/overlay/detail/PeerImp.cpp

```diff
@@ -1275,7 +1273,7 @@ PeerImp::handleTransaction(
         {
             JLOG(p_journal_.warn()) << "Ignoring Network relayed Tx containing "
                                        "tfInnerBatchTxn (handleTransaction).";
-            fee_ = Resource::feeHighBurdenPeer;
+            fee_.update(Resource::feeModerateBurdenPeer, "inner batch txn");
```

Comment:
why the change here?

Reply:
I changed it because I'm not sure its a HeavyBurdenPeer. Open to hearing a justification if it should be heavy or moderate?

```
Charge const feeModerateBurdenPeer(250, "moderate peer request");
Charge const feeHeavyBurdenPeer(2000, "heavy peer request");
```

Reply:
anyways i don't think it should matter...inner batch will never be relayed anyways

Reply:
If this code is untouchable, can it be exempt from codecov?

Reply:
Same with below

Reply:
fixed: https://github.com/XRPLF/rippled/pull/5060/commits/8a39c03667a4fb69c36e72a9e5282b88388df8de